docs/user_guide/ug_general_concepts.md

```diff
@@ -1 +1,42 @@
-TODO
\ No newline at end of file
+# General concepts
+
+MechaPhlowers is a library specialized to perform mechanical and geometrical calculus for powerlines. The models are designed to be the simplest and most accurate possible : when possible, calculus are performed in 2D planes.
```

Comment:
MechaPhlowers is a library specialized in performing mechanical and geometrical calculations for power lines. The models are designed to be as simple and accurate as possible: whenever appropriate, calculations are performed in 2D planes.
"powerline" collé c'est une marque déposée !